app/components/cases/cases-header.js

```diff
@@ -5,8 +5,6 @@ import { action } from '@ember/object';
 export default class CasesHeader extends Component {
```

Comment:
Kan je eens bekijken wat het geeft als je `import Component from '@ember/component';` vervangt door `mport Component from '@glimmer/component';`? Hierdoor zal een wrapping div verdwijnen. Even  hecken of dan nog alles op z'n plaats zit. (de `this.toggleProperty` zal vervangen moeten worden en `isAddingCase` zal `tracked` moeten worden. Geef gerust een seintje daarvoor.